CHANGELOG.next.toml

```
@@ -10,3 +10,9 @@
 # references = ["smithy-rs#920"]
 # meta = { "breaking" = false, "tada" = false, "bug" = false }
 # author = "rcoh"
+
+[[aws-sdk-rust]]
+message = "Fix dev-dependency cycle between aws-sdk-sso and aws-config"
+meta = { "breaking" = false, "tada" = false, "bug" = true }
+references = ["smithy-rs#1088"]
```

Not sure if pointing at the 0.35 release was intentional.
```suggestion
references = ["smithy-rs#1089"]
```

---

no I just guessed at the next id 😅